### ISSUE TITLE
Crypto-Restart-1.5k-12m and ScheduleFocus-Restart-1.5-12m tests failed using 0.15.0-alpha.1 sdk

### ISSUE THREAD

**Issue by Neeharika-Sompalli** [hashgraph/hedera-services#1513]
Crypto-Restart-1.5k-12m and ScheduleFocus-Restart-1.5-12m tests failed using 0.15.0-alpha.1 sdk . The errors include `SleepCheckException` and `Freeze` stage did not start in the test. 

Slack Summary: https://hedera-hashgraph.slack.com/archives/C018Y4E6ADT/p1622531415032600

**UPDATE**
Passed test links are updated in the below comment. Will be closed once PR is opened.

**Comment by Neeharika-Sompalli** [comment]
RestartRun has the following comment in the runConfig
```
// currently only services restart tests set freeze time by modifying settings
		// because services is using a sdk which doesn't contain freeze changes
		// Todo: when integrating sdk with freeze changes into services,
		//  we need to remove the following code, and add `genesisFreezeAfterSecs` config to services restart
		//  tests json configs
```

**Comment by Neeharika-Sompalli** [comment]
Removing the conditional code added for services to support old freeze time, and adding `genesisFreezeAfterSecs=300` resolved restart issues. RestartValidator still fails because of the ` ERROR EXCEPTION        <<FCHashMap: garbage-collector ? #0>> FCHashMapGarbageCollector: FCHashMap garbage collection queue size exceeds threshold {"queueSize":102} [com.swirlds.logging.payloads.GarbageCollectionQueuePayload]` . 
This will be resolved when new `0.15.0-alpha.2` sdk , when the setting to increase garbage collection queue size can be increased.

**Comment by Neeharika-Sompalli** [comment]
Passing Restart tests : 
1. Crypto-Restart-1.5k-12m : https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1622665687207000
